purchase.adoc

```diff
@@ -388,288 +517,533 @@ Fields named "*UUID1" are the standard UUID Type 1 (8-4-4-4-12) representation o
   "payments": [
     {
-      "uuid": "165b88a0-07a3-11e6-9dae-43c30f1bff5b",
-      "amount": 1500,
-      "gratuityAmount": 0,
-      "type": "IZETTLE_CARD",
-      "attributes": {
-        "transactionStatusInformation": "EC00",
-        "cardPaymentEntryMode": "EMV",
-        "maskedPan": "476173******0119",
-        "installmentAmount": null,
-        "referenceNumber": "AU54FYHW7X",
-        "nrOfInstallments": 0,
-        "cardType": "VISA",
-        "terminalVerificationResults": "0080088000",
-        "applicationIdentifier": "A0000000031010",
-        "applicationName": "Visa Credit"
-      }
+      "uuid": "592b82f2-3ee4-11eb-9f71-a3ad065a16fc",
+      "amount": 70000,
+      "type": "SWISH",
+      "attributes": {}
     }
   ],
-  "cashRegister": {
-    "uuid": "r_dBcr6tT2ypC1jlMuUUzg",
-    "displayName": "Kassa 1"
-  },
   "receiptCopyAllowed": true,
-  "refund": false,
+  "references": {
+    "checkoutUUID": "589c2de6-3ee4-11eb-9f71-a3ad065a16fc"
+  },
+  "created": "2020-12-15T14:46:43.592+0000",
+  "refunded": false,
   "groupedVatAmounts": {
-    "12.0": 1500
+    "25.0": 70000
   },
-  "refunded": false
+  "refund": false
 }
-```
-
-### Products
+----
+--
 
-A purchase has one or more item lines represented in the `products` list attribute. The `type` attribute indicates what
-kind of item line the entity represent, with `PRODUCT`, `CUSTOM_AMOUNT` and `GIFTCARD` being the currently available types.
+'''
+====
 
-In parallell to the `type` attribute there is an optional `details` attribute that may contain information related to the specific type,
-for example item lines of type `GIFTCARD` will have a `giftcardUuid` attribute pointing out what giftcard that was sold/returned as part of purchases
+==== Response Model
 
-### Payments
+===== Purchase
+Purchase represents a receipt, it's immutable and contains information about items that were purchased, one or several payments received, discounts applied etc.
+Refunds are represented by the same model but with negative amounts and refund related flags/fields. +
+*Note*: More attributes can potentially be added, please make sure your integration is build in a tolerant way.
 
-A purchase has one or more payments represented in the `payments` list attribute.
+*Attributes*
+[cols="15%,10%,75%"]
+|===
 
-=== Payment attributes
-The `payment` data type only contain a few generic parameters such as `uuid`, `amount` and `type`.
+|Name|Type|Description
 
-The currently available payment types are:
+|source|string a|Source of purchase, value can be one of the following
 
-- `IZETTLE_CARD`
-- `IZETTLE_CARD_ONLINE`
-- `IZETTLE_CASH`
-- `IZETTLE_INVOICE`
-- `SWISH`
-- `VIPPS`
-- `MOBILE_PAY`
-- `PAYPAL`
-- `STORE_CREDIT`
-- `GIFTCARD`
-- `KLARNA`
+* "POS" - purchase made with iZettle GO app
+* "WEB_SHOP" - purchase made online
+|purchaseUUID|string|Unique id of a purchase within iZettle system.
+|purchaseUUID1|string|Unique id of a purchase within iZettle system as UUID version 1.
```

Comment:
Is there a difference between `purchaseUUID1` and `purchaseUUID`?

Reply:
yes

Reply:
Should we even state that the former id deprecated?

Reply:
Sure, even better 👍

Reply:
I'd also argue that we should even remove " within iZettle system". It is actually *universally unique*.